js_modules/dagster-ui/packages/ui-core/src/app/AppProvider.tsx

```diff
@@ -52,10 +52,13 @@ export interface AppProviderProps {
     telemetryEnabled?: boolean;
     statusPolling: Set<DeploymentStatusType>;
   };
+
+  // Used for localStorage/IndexedDB caching to be isolated between instances/deployments
+  localCacheIdPrefix?: string;
```

Comment:
This seems awesome but I wonder if we should name it more generically? If it's an instance / deployment ID, it feels like we'll want to use it for other things in the future too

Reply:
i thought this was pretty general hut open to suggestions